clang/lib/CIR/CodeGen/CIRGenTypeCache.h

```diff
@@ -107,7 +108,7 @@ struct CIRGenTypeCache {
   //     unsigned char SizeAlignInBytes;
   //   };
 
-  //   clang::LangAS ASTAllocaAddressSpace;
+  clang::LangAS ASTAllocaAddressSpace;
```

Comment:
Seems like this needs to be initialized to `clang::LangAS::Default`?

Reply:
I am following the exact pattern of clang’s`CodeGenTypeCache`: this value is initialized in `CIRGenModule` constructor using the factory around `TargetCIRGenInfo` base class.

Reply:
I am following the exact pattern of clang’s`CodeGenTypeCache`: this value is initialized in `CIRGenModule` constructor using the factory around `TargetCIRGenInfo` base class.

Reply:
Oh, I see that now. Thanks